e2e/panels-suite/frontend-sandbox-panel.spec.ts

```diff
@@ -85,9 +99,23 @@ describe('Panel sandbox', () => {
     it('Does not reaches out of panel div', () => {
       // this button reaches out of the panel div and modifies the element dataset
       cy.get('[data-testid="button-reach-out"]').click();
-
+      cy.wait(100); // small delay to prevent false positives from too fast tests
       cy.get('[data-sandbox-test="true"]').should('not.exist');
     });
+
+    it('Does not Reaches out of the panel editor', () => {
+      e2e.flows.openDashboard({
+        uid: DASHBOARD_ID,
+        queryParams: {
+          '__feature.pluginsFrontendSandbox': false,
+          editPanel: 1,
+        },
+      });
+
+      cy.get('[data-testid="panel-editor-custom-editor-input"]').type('x');
+      cy.wait(100); // small delay to prevent false positives from too fast tests
+      cy.get('[data-sandbox-test="panel-editor"]').should('not.exist');
+    });
   });
 
   afterEach(() => {
```

Comment:
Not sure what the revert command does, but you probably want to do this in before each instead as there's no guarantee this code is ever executed.

Reply:
I am almost sure I  copied this pattern from other e2e tests doing the same.